CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   resources would error out. This manifested as the `api-checks-pod` ending up in
   a `CrashLoopBackOff`. More information can be found in the related
   [bug](https://bugzilla.redhat.com/show_bug.cgi?id=2117268)
+- When the scan settings set the `debug` attribute to `true`, CO wasn't
+  cleaning the scan pods properly when the scan was deleted. This
```

Comment:
nit: s/cleaning/deleting/

Only because cleaning could mean different things to different readers.

Reply:
fixed, thanks